### PR TITLE
test: remove timeout for puppeteer waitForSelector

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

总是有一两个用例挂掉：https://app.argos-ci.com/ant-design/ant-design/builds/11241/57066052
试试 https://pptr.dev/api/puppeteer.page.waitforselector#remarks

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    --       |
| 🇨🇳 Chinese |   --        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 4d470c1</samp>

Increased the timeout option for `waitForSelector` in `imageTest.tsx` to avoid test failures. This was part of a pull request to improve the test coverage and reliability of the ant-design components.

### <samp>🤖 Generated by Copilot at 4d470c1</samp>

* Increase the timeout option for `waitForSelector` method in image snapshot test to 100000 milliseconds ([link](https://github.com/ant-design/ant-design/pull/44611/files?diff=unified&w=0#diff-34d634572b80c540ebecd7780004e6031fbb9f004387e3df7302e346940ffae4L78-R80)). This improves the stability of the test, which sometimes failed due to the selector not being found in time.

### CODE REVIEW

**Comment by afc163** [comment]
没用。。。

File: package.json [hunk 1]
```diff
@@ -198,7 +198,6 @@
     "@types/node": "^20.0.0",
     "@types/prismjs": "^1.26.0",
     "@types/progress": "^2.0.5",
-    "@types/puppeteer": "^7.0.4",
```
Comment (afc163) [inline package.json:201]: https://www.npmjs.com/package/@types/puppeteer 已经不再需要了。

File: package.json [hunk 2]
```diff
@@ -194,11 +194,9 @@
     "@types/jest-image-snapshot": "^6.1.0",
     "@types/jquery": "^3.5.14",
     "@types/lodash": "^4.14.139",
-    "@types/lz-string": "^1.3.34",
```
Comment (afc163) [inline package.json:197]: https://www.npmjs.com/package/@types/lz-string 已经不再需要了。

File: package.json (hunk 1 above)
Comment (afc163) [reply package.json:201]: puppeteer 居然是被 @types/puppeteer 带进来的。。。

**Review by kiner-tang — APPROVED** [review]

**Review by li-jia-nan — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Review by MadCcc — APPROVED** [review]

File: tests/shared/imageTest.tsx [hunk 3]
```diff
@@ -75,16 +75,20 @@ export default function imageTest(component: React.ReactElement) {
...
     );
 
-    await page.waitForSelector('#end-of-screen');
+    await page.waitForSelector('#end-of-screen', {
+      timeout: 0,
+    });
 
     const image = await page.screenshot({
       fullPage: true,
+      captureBeyondViewport: true,
```
Comment (MadCcc) [inline tests/shared/imageTest.tsx:84]: 这个默认就是 true 吧

**Pull request merged**